modules/cbQuestion/language/en_us.lang.php

```diff
@@ -112,5 +112,6 @@
 	'Export Results' => 'Export Results',
 	'Value' => 'Value',
 	'querytype' => 'Query Type',
+	'Direct Sql' => 'Direct Sql',
```

Comment:
you must add this to all the .php language files in that directory, not just this one. add the line in English, you don't have to translate it but do put it